style.css

```diff
@@ -159,6 +172,16 @@
 
 /* media queries */
 
+@media screen and (max-width: 1250px) {
+  .nav-center input {
+    display: none;
+  }
+
+  .grid-col-3 {
+    grid-template-columns: 1fr 1fr;
+  }
```

Comment:
Suggestion: Classname is `grid-col-3`, so the property should also specify 3 columns of 1fr each.

Reply:
Thanks for this. Actually for media query it changes. Noting this down for further refactoring.